platform/darwin/src/MGLSymbolStyleLayer.h

```diff
@@ -1204,6 +1206,15 @@ MGL_EXPORT
  * `MGLCameraStyleFunction` with an interpolation mode of:
    * `MGLInterpolationModeExponential`
    * `MGLInterpolationModeInterval`
+ * `MGLSourceStyleFunction` with an interpolation mode of:
```

Comment:
Yes, if you rebase this PR on top of master as of now, f901e77 will fix this problem 👍